manager/volume.go

```diff
@@ -49,6 +51,44 @@ func (m *VolumeManager) List() (map[string]*longhorn.Volume, error) {
 	return m.ds.ListVolumes()
 }
 
+// sortKeys accepts a map with string keys and returns a sorted slice of keys
+func sortKeys(mapObj interface{}) ([]string, error) {
+	if mapObj == nil {
+		return []string{}, errors.New("mapObj was nil")
```

Comment:
Use `fmt.Errorf` instead. It's an item checked by go report (https://goreportcard.com/report/github.com/rancher/longhorn-manager)

Also cry `BUG: `, since it shouldn't be possible for the caller to pass non-map or nil.

Same below.